### DISCUSSION TITLE
How to adjust the time step size for the MRT

### DISCUSSION THREAD

**Discussion by ztdepztdep** [CFD-GO/TCLB#369] [Q&A]
can we adjust the time step size to increase the stability of the MRT.

**Comment by TravisMitchell** [comment]
@ztdepztdep can you please place questions like this into Discussions. They are not **Issues** with the codebase.

**Comment by llaniewski** [comment]
@ztdepztdep timestep in LBM is always 1. You can read up on transferring LBM unitless values into real-world units in the tutorial on LBM: https://docs.tclb.io/tutorials/model-development/2.-d2q9-single-relexation-time/
